### PR TITLE
Problem: no place for a proxy settings file

### PR DESCRIPTION
**Pull request by jimklimov**
Solution: prepare an /etc/default/fty-proxy with presumably sufficient access rights

Signed-off-by: Jim Klimov <EvgenyKlimov@eaton.com>

### CODE REVIEW

**Comment by geraldguillaume** [comment]
Please @jimklimov have a look to the conflict

File: systemd/fty-proxy.service.in [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Prepare common run-time environment variables for 42ity services
```
Comment (jana-rapava) [inline systemd/fty-proxy.service.in]: I think this description should be updated.

Comment (jimklimov) [reply systemd/fty-proxy.service.in]: Yes :)

**Comment by jimklimov** [comment]
2 comments addressed

**Pull request merged**